.travis.yml

```diff
@@ -10,6 +10,8 @@ node_js:
 - 18
 
 before_install:
+- echo -e "machine github.ibm.com\n  login $GITHUB_ACCESS_TOKEN" > ~/.netrc
```

Comment:
Remove this echo stmt as per the template comment

Reply:
Removed